### ISSUE TITLE
Parts of UI not working and Steam Login Errors

### ISSUE THREAD

**Issue by RaiuTheEevee** [hippogamesunity/PixelStudioHub#160]
This is a multi-part issue that may or may not all be related to each other. Also this is on the steam version of the software, I haven't tested this on the android version yet.

I was going to check out the shop to purchase a pro license because I've really been liking this software. I also got the app on my phone, and wanted to get pro cross so I could sync them. When I went to the shop I got the error "/Client/LoginWithSteam: EvaluationModePlayerCountExceeded". I also was unable to click on certain parts of the menu, which I circled in red in the first screenshot below.

I tried to sign into the pixel network through steam and I got the same error, so I instead created an account with the pixel network. That worked so I went to my profile, and I started having a similar issue with the ui in my profile. I basically wasn't able to click on anything on this screen except for the parts that I circled in red in the second screenshot below. Basically the only way to exit my profile was to press the escape key.

The problem with my profile eventually fixed itself, I have no idea how, but I still can't access the purchase options in the shop; it's the same ui issue as before. Furthermore, after signing into the pixel network, I get the same error message when trying to link my steam account. I also still get the same steam login error when opening the shop menu, but now half the time I get this other error instead which I was able to capture in the first screenshot.

![image_2023-01-18_191135159](https://user-images.githubusercontent.com/102767817/213325344-eea2d846-6b28-4fb4-a4f3-be715d327779.png)
![image_2023-01-18_191456362](https://user-images.githubusercontent.com/102767817/213325730-63db1deb-21ce-4316-9307-3cc3b6084ef3.png)

**Comment by RaiuTheEevee** [comment]
Just a side note, I was able to make my purchase on the android version and sync my projects.

**Comment by hippogamesunity** [comment]
Hello, thanks for reporting!

**Comment by hippogamesunity** [comment]
@RaiuTheEevee can you please try again and check if the issue is fixed or not?

**Comment by hippogamesunity** [comment]
Just Sign-in with Steam in Pixel Network

**Comment by RaiuTheEevee** [comment]
@hippogamesunity 
It looks fixed to me! I signed out to see if the shop was working and it was! The prices showed up and clicking on them opened the transaction page. Clicking the Restore button also gave the correct message of "Purchases not found" because I'm not signed in. The only thing that seems to still not be working here is the "?" under the regular Pro purchase button. Clicking it doesn't do anything.

I was able to sign in with steam, and I was able to link my steam with my pixel network account! No error messages anywhere!
